lbrynet/core/BlobManager.py

```diff
@@ -70,6 +70,12 @@ def blob_paid_for(self, blob_hash, amount):
     def get_all_verified_blobs(self):
         pass
 
+    def add_blob_to_download_history(self, blob_hash, host, rate):
```

Comment:
Nothing at the moment. It could give counts for what blobs you've uploaded, and historical blob rates could be used in a price model.